integr8ly/ci-cd
# created installation-smoke-tests.yaml jjb

## Motivation
https://issues.jboss.org/browse/INTLY-561

## What
Create jjb for clean-uninstallation-pipeline

## Verification Steps
Check is to run this command inside the repository with PR changes to confirm that the update is successful (and also make sure that the pipeline configuration does not change -> as the template is the same as definition on Jenkins): 

`jenkins-jobs --conf jenkins_jobs.ini update jobs/integr8ly/installation-smoke-tests.yaml`

## Checklist:


## Progress
